crates/biome_js_formatter/tests/specs/js/module/declarations/variable_declaration.js.snap

```diff
@@ -655,9 +655,11 @@ let loooooooooooooooooooooooooong3 = objdddddddddectobjddsadsaddddddddect(
 var loooooooooooooooooooooooooong1 =
 	fnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfnfn(`123123
  dsa`);
-let loooooooooooooooooooooooooong2 = objdddddddddectobjdddsadsadddddddect(`123123
+let loooooooooooooooooooooooooong2 =
+	objdddddddddectobjdddsadsadddddddect(`123123
 dsa`).ewqeqewqweqweqweqweqweqweqw;
-let loooooooooooooooooooooooooong3 = objdddddddddectobjddsadsaddddddddect(`123123
+let loooooooooooooooooooooooooong3 =
+	objdddddddddectobjddsadsaddddddddect(`123123
```

Comment:
This formatting update is reasonable as it matches Prettier's behavior. [playground link](https://biomejs.dev/playground/?code=bABlAHQAIABsAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBuAGcAMgAgAD0AIABvAGIAagBkAGQAZABkAGQAZABkAGQAZABlAGMAdABvAGIAagBkAGQAZABzAGEAZABzAGEAZABkAGQAZABkAGQAZABlAGMAdAAoAGAAMQAyADMAMQAyADMACgBkAHMAYQBgACkALgBlAHcAcQBlAHEAZQB3AHEAdwBlAHEAdwBlAHEAdwBlAHEAdwBlAHEAdwBlAHEAdwBlAHEAdwA7AAoAbABlAHQAIABsAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBvAG8AbwBuAGcAMwAgAD0AIABvAGIAagBkAGQAZABkAGQAZABkAGQAZABlAGMAdABvAGIAagBkAGQAcwBhAGQAcwBhAGQAZABkAGQAZABkAGQAZABlAGMAdAAoAGAAMQAyADMAMQAyADMACgAgAGQAcwBhAGAAKQBbAGQAcwBhAGQAcwBhAGQAcwBhAGQAcwBhAGQAcwBhAGQAcwBhAGQAcwBhAF0AKAApAC4AZQB3AHEAbwBlAHcAcQBvAGUAaQBvAHcAcQBpAGUAbwBwAHcAcQBpAGUAOwA%3D)